instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/OpenTelemetryAutoConfiguration.java

```diff
@@ -83,16 +82,14 @@ public SdkLoggerProvider sdkLoggerProvider(
       SdkLoggerProviderBuilder loggerProviderBuilder = SdkLoggerProvider.builder();
       loggerProviderBuilder.setResource(otelResource);
 
-      loggerExportersProvider.getIfAvailable(Collections::emptyList).stream()
+      loggerExportersProvider
+          .getIfAvailable(Collections::emptyList)
           .forEach(
               loggerExporter ->
                   loggerProviderBuilder.addLogRecordProcessor(
                       BatchLogRecordProcessor.builder(loggerExporter).build()));
 
-      SdkLoggerProvider loggerProvider = loggerProviderBuilder.build();
-      GlobalLoggerProvider.set(loggerProvider);
-
-      return loggerProvider;
+      return loggerProviderBuilder.build();
```

Comment:
The Spring starter does not set `GlobalOpenTelemetry`, we'll have to do that if we want logs to work (and add `resetForTests()` calls to the test classes)

Reply:
thx, I totally missed this 🙈